src/runtime/crt/Makefile

```diff
@@ -0,0 +1,57 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Given that we are going to have StandaloneCrt.cmake, can we use CMake to build this instead?

Reply:
Im not sure how to do this without dragging in extra tvm configuration from e.g. include_directories() calls. The other thing is that the make-based flow is a good example for firmware engineers not used to cmake. What do you think? Cc @tqchen

Reply:
I will let you guys decide, but it might make sense to show a make flow if it is  simple enough, because others can learn from it